include/exadg/operators/inverse_mass_operator.h

```diff
@@ -277,7 +277,8 @@ class InverseMassOperatorHdiv
     {
       preconditioner =
         std::make_shared<JacobiPreconditioner<MassOperator<dim, n_components, Number>>>(
-          mass_operator);
+          mass_operator, true);
```

Comment:
'/* initialize_preconditioners */' ?

Reply:
I am somewhat undecided. I  can change it if you want. If so, we should do it consistently (i.e. there are probably also other places).

Reply:
we can do it or not do it, I do not think this is too strict ... but in line 139 above you had it and here in line 280 its not. So decide for one variant and make it consistent in the file :smile: